conn_go18.go

```diff
@@ -126,3 +127,63 @@ func (cn *conn) cancel() error {
 		return err
 	}
 }
+
+var defaultConverter = &converter{}
+
+func (st *stmt) ColumnConverter(idx int) driver.ValueConverter {
+	return defaultConverter
+}
+
+type converter struct{}
+
+func (c *converter) ConvertValue(v interface{}) (driver.Value, error) {
+	if driver.IsValue(v) {
+		return v, nil
+	}
+
+	switch value := v.(type) {
+	case int:
+		return int64(value), nil
+	case int8:
+		return int64(value), nil
+	case int16:
+		return int64(value), nil
+	case int32:
+		return int64(value), nil
+	case int64:
+		return int64(value), nil
+	case uint:
+		return int64(value), nil
+	case uint8:
+		return int64(value), nil
+	case uint16:
+		return int64(value), nil
+	case uint32:
+		return int64(value), nil
+	case uint64:
+		if value >= 1<<63 {
```

Comment:
Sorry, I've only now noticed, but you could use `value > math.MaxInt64` here.

Reply:
Done